src/App.jsx

```diff
@@ -13,6 +13,7 @@ import { UserContext, UserProvider } from './components/UserProvider';
       <NavBar />
       <Routes>
         <Route path="/" element={<Articles />} />
+        <Route path="/:topic" element={<Articles />} />
```

Comment:
It could be a bit more clear if this path was tweaked into being topics/:topic, as then the url will tell the user what the parametric part actually is